### DISCUSSION TITLE
Audio from device

### DISCUSSION THREAD

**Discussion by tazione** [DISTRHO/Cardinal#440] [General]
Hi all, I just only want to know if exist an audio module able to make some routing like the audio8 on vcv here in the attachment.
Thanks a lot, I love Cardinal!

![Cattura](https://user-images.githubusercontent.com/15073139/204477137-1d7eb485-945d-41d8-8390-6307b2d5f470.PNG)

**Comment by falkTX** [comment]
Intentionally not. You can use the CardinalNative binary/build which provides that in a way, through the default audio device on the system.

Cardinal main focus is being a plugin, the standalone is still a side thing that does not get priority.

**Comment by tazione** [comment]
Got it.
Thanks a lot!

Il Mar 29 Nov 2022, 09:40 Filipe Coelho ***@***.***> ha
scritto:
